### DISCUSSION TITLE
blog/javascript/2-var-let-const

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#50] [General]
# blog/javascript/2-var-let-const

변수 선언 키워드인 var, let, const의 차이를 스코프와 호이스팅의 관점에서 다뤄보겠습니다.

https://www.enjoydev.life/blog/javascript/2-var-let-const

**Comment by twkim8548** [comment]
안녕하세요, 글 너무 이해 잘되게 작성해주셔서 기초 지식 복습 겸 잘 읽고 있습니다. 
항상 기록하는걸 습관화 해보고 싶어도 잘 되지 않는데 이렇게 조리있게 잘 쓰시니 멋지십니다👍 

마지막에 작성해주신 let/const 선언문 호이스팅과 함수 표현식 호이스팅에서 var이 아닌 let/const로 선언했을때의 오류 예시를 작성해주신게 아닐까 싶은데, var로 선언되어있네요! 

제가 의도하신바를 잘못 이해했을 수도 있겠지만, 혹시나 하여 댓글 남겨봅니다 :)

**Reply by pySoo** [reply]
안녕하세요!
개념들을 소소하게 정리해본 글인데 도움이 되었다니 정말 기쁘네요 ☺️

짚어주신 부분 let/const로 선언이 되었어야 했는데 제가 var로 선언해두었군요 수정했습니다!
꼼꼼하게 읽어주시고 댓글로 알려주셔서 너무 감사드려요 👍👍
